input/pagecontent/group-genomics.md

```diff
@@ -1,21 +1,94 @@
+
 The identity of non-genomic laboratory tests is typically represented by a [Logical Observation Identifiers and Names (LOINC)](https://loinc.org/) code. However, many genomic tests and panels do not have LOINC codes, although some might have an identifier in the [NCBI Genomic Testing Registry (GTR)](https://www.ncbi.nlm.nih.gov/gtr/), a central location for voluntary submission of genomic test information by providers. While GTR is a viable source for identifying many genomic tests, the user should be aware that the GTR is not single authoritative source since the test data is voluntarily updated. Standardization of codes for genomic tests is essential to facilitate data analysis of genomic tests, and should be a priority for the genomics testing community in the near future. Implementers should also note that, to conform to the requirements of the [US Core Laboratory Result Profile](http://hl7.org/fhir/us/core/StructureDefinition-us-core-observation-lab.html), if a suitable LOINC exists, it must be used. If there is no suitable code in LOINC, then a code from an alternative code system such as GTR can be used.
 
-### Representing Fusion Genes
+### Representing Variants
+
+#### Variant Nomenclature
+
+The [HL7 GRIG guidance on defining variants](http://hl7.org/fhir/uv/genomics-reporting/sequencing.html) states support for two reporting patterns for specifying variants:
+1. by HGVS or ISCN nomenclature
+2. by providing multiple component details in the style of VCF columns
+
+Although mCODE should support both formats, it is preferable to use HGVS nomenclature for coding and protein variants where possible.
+
+A full representation of an HGVS variant should include the following: 
+* a transcript identifier or reference sequence id that includes the version number followed by a nomenclature which further characterizes the nature of the variant.
+* the reference sequence assembly version populated in the Variant observation component [reference-sequence-assembly](http://hl7.org/fhir/uv/genomics-reporting/STU2/StructureDefinition-finding-definitions.html#Observation.component:reference-sequence-assembly).
+
+**NOTE**: The FHIR IG publisher uses the [Mutalyzer 3](https://mutalyzer.nl/) Application Programming Interface (API) for validating an HGVS-notated variant included in the FHIR examples. It is not intended to validate all possible variants as there is no single HGVS validation tool that comprehensively covers the most up-to-date recognized variants for a given reference sequence. 
+
+Every FHIR genomics resource referenced by a genomics report (e.g.: Variant, Diagnostic Implication, Therapeutic Implication), should include a date or datetime, even if there was only one date included in the genomic report for all results. Doing so simplifies the ability for the implementer to directly track changes in cases of a variant reclassification or reinterpretation. 
```

Comment:
Will remove the reference to Mutalyzer, especially since Grahame wants to migrate the IG Publisher HGVS validation code to use the [LHC FHIR CodeSystem Operations for HGVS](https://clinicaltables.nlm.nih.gov/hgvs.html). 
The point was to note that the FHIR IG examples might have errors because the validator that the IG publisher uses does not recognize all possible variants.